app/src/components/Modals/add-activity-modal.tsx

```diff
@@ -0,0 +1,440 @@
+"use client";
```

Comment:
What is the difference between add-activity-modal and add-activity-energy-consumption-modal?
We don't want to implement a specific React component for each type of methodology in the future, so it would be good to dynamically construct the form from the `InputMethodology` data once we have that.